src/components/Button.js

```diff
@@ -1,9 +1,46 @@
 import React from 'react';
 
-const ButtonSearch = (props) => (
- <button className="btn btn-primary fn-submit-name">
-     {props.name} {props.id}
-    </button>
-)
+class Button extends React.Component {
```

Comment:
It looks like the purpose of this component has changed - it no longer only renders a button. So if I was trying to use `<Button />` then I would be a bit confused. I think the best solution here would be to rename this component to reflect the job that it's actually doing. Can you think of a good component name?